### PR TITLE
feat: add native app compare docs

### PR DESCRIPTION
**Pull request by wswebcreation**
## Proposed changes

This PR adds docs adjustments for supporting native apps with the visual service

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: website/blog/2024-02-05-visual-testing-enhancements.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+We're excited to announce an update to the Visual Testing Module with a key focus: **Native App Snapshot Testing**. Our vision has always been to provide a comprehensive, versatile testing tool that simplifies your workflow. This update is a step towards creating a 'Swiss Army Knife' for visual testing, catering to diverse requirements across platforms and extending our support to native mobile applications, making your testing process more efficient and seamless.
+
+#### Native App Snapshot Testing
+
+The module now supports `saveElement`, `checkElement`, `saveScreen`, and `checkScreen` methods, along with `toMatchElementSnapshot` and `toMatchScreenSnapshot` matchers for native apps. It automatically detects the testing context (web, webview, or native_app) to streamline your workflow.
+
+#### Key Changes and Improvements
+
+- **`resizeDimensions` Update**: Now exclusively accepts an object format.
+- **Native App Context**: Methods like `saveFullPageScreen` and `checkTabbablePage` are not compatible with native mobile apps.
```
Comment (christian-bromann) [inline website/blog/2024-02-05-visual-testing-enhancements.md:17]:
```suggestion
- **Native App Context**: Methods like `saveFullPageScreen` and `checkTabbablePage` are now compatible with native mobile apps.
```

File: website/blog/2024-02-05-visual-testing-enhancements.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+#### Special Thanks to wswebcreation
+
+We owe a big thank you to [wswebcreation](https://github.com/wswebcreation) for his work in [`wdio-native-app-compare`](https://github.com/wswebcreation/wdio-native-app-compare), which inspired this enhancement. His contribution has been vital in advancing our module's capabilities.
+
+Thank you for your continued support, and we look forward to your feedback on these new features.
+
+Happy testing!
+
+*The WebdriverIO Team*
```
Comment (christian-bromann) [inline website/blog/2024-02-05-visual-testing-enhancements.md:29]: Let's change the author above to "WebdriverIO Team" and update twitter handle and image url.

Comment (wswebcreation) [reply website/blog/2024-02-05-visual-testing-enhancements.md:29]: Will do that

**Review by christian-bromann — APPROVED** [review]
So much great news 🎉

**Pull request merged**